src/main/resources/application.yml

```diff
@@ -149,8 +149,7 @@ spring:
     #      starttls:
     #        enable: true
     #        required: true
-  profiles:
-    active: default
+  profiles: default, test, dev
```

Comment:
README.md was lying to me. While rerunning commands it no longer worked to specify profile external property.